homeassistant/components/ipp/const.py

```diff
@@ -0,0 +1,25 @@
+"""Constants for the IPP integration."""
+
+# Integration domain
+DOMAIN = "ipp"
+
+# Attributes
+ATTR_IDENTIFIERS = "identifiers"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MARKER_TYPE = "marker_type"
+ATTR_MARKER_LOW_LEVEL = "marker_low_level"
+ATTR_MARKER_HIGH_LEVEL = "marker_high_level"
+ATTR_MODEL = "model"
+ATTR_PRINTER_COMMAND_SET = "printer_command_set"
+ATTR_PRINTER_INFO = "printer_info"
+ATTR_PRINTER_LOCATION = "printer_location"
+ATTR_PRINTER_SERIAL = "printer_serial"
+ATTR_PRINTER_STATE_MESSAGE = "printer_state_message"
+ATTR_PRINTER_STATE_REASON = "printer_state_reason"
+ATTR_PRINTER_URI_SUPPORTED = "printer_uri_supported"
```

Comment:
Why prefix with `printer_`. It's attributes for a printer entity, isn't it all printer info ?

Reply:
removed prefix